### PR TITLE
Trading View: Fix: Switch to orders tab after placing an order

### LINKED ISSUE #1821: [BUG]: app does not switch to Orders tab any more after placing an order
previous versions switched to "Orders" tab after an order was placed, so user could see the status, like this
image

now, after placing the order, app shows this warning
```
[13:24:39] [warning] [main.prerequisites.hpp:92] [408623]: qrc:/Dex/Exchange/Trade/ProView.qml:102: ReferenceError: tabView is not defined
```
and stays on "Chart" tab

### PR DESCRIPTION
**Pull request by SylEze**
Closes #1821

### CODE REVIEW

**Comment by cipig** [comment]
it doesn't switch to Orders tab after order placement, it stays on Chart, but the warning is gone

**Comment by SylEze** [comment]
@cipig should be fixed this time ^^

**Comment by cipig** [comment]
that is fixed, it now switches to Order tab after placing order
if i then switch pairs it stays in Order tab though... is that intended or was it intended to switch back to Chart when you change pairs?
if i leave DEX, go to Wallet or Portfolio and then back to DEX it defaults to Chart tab

**Review by cipig — DISMISSED** [review]

**Comment by SylEze** [comment]
> if i then change to another pair it stays in Order tab though... is that intended or was it intended to switch back to Chart when you change pairs?

I don't know if it was like that before but this behaviour is not specified anywhere in the code

**Comment by cipig** [comment]
> I don't know if it was like that before but this behaviour is not specified anywhere in the code

There is no "before" because Chart was not in a tab before, and after it was, the switching to Orders was not done :-)
It's perfectly fine, let's merge this.

File: atomic_defi_design/Dex/Exchange/ProView/TradingInfo/Main.qml [hunk 1]
```diff
@@ -12,18 +12,13 @@ import "../../ProView"
...
 Widget
 {
+    property alias currentIndex: tabView.currentIndex
+
     title: qsTr("Trading Information")
 
     background: null
     margins: 0
 
-    Connections
```
[the remark this answers is not in the document]
Comment (SylEze) [reply atomic_defi_design/Dex/Exchange/ProView/TradingInfo/Main.qml:20]: Those events are triggerred when an order is placed, that's why I removed it, it conflicts with the goal of this PR

But I pushed another solution @smk762

**Review by smk762 — APPROVED** [review]

**Review by cipig — APPROVED** [review]

**Pull request merged**